index.js

```diff
@@ -28,7 +29,14 @@ function defaultKey (req, file, cb) {
 function autoContentType (req, file, cb) {
   file.stream.once('data', function (firstChunk) {
     var type = fileType(firstChunk)
-    var mime = (type === null ? 'application/octet-stream' : type.mime)
+    var mime = 'application/octet-stream'
+
+    if (type === null && isSvg(firstChunk)) {
+      mime = 'image/svg+xml'
+    } else {
+      mime = type.mime
```

Comment:
As I read it this will be triggered if `type` is `null` and the `firstChunk` doesn't look like an svg...

How about

```js
var mime

if (type) {
  mime = type.mime
} else if (isSvg(firstChunk)) {
  mime = 'image/svg+xml'
} else {
  mime = 'application/octet-stream'
}
```